hadoop-hdds/interface-admin/src/main/proto/ScmAdminProtocol.proto

```diff
@@ -529,7 +529,8 @@ message ContainerTokenSecretProto {
     required string ownerId = 1;
     required ContainerID containerId = 2;
     required uint64 expiryDate = 3;
-    required string certSerialId = 4;
+    optional string certSerialId = 4 [deprecated=true];
+    required UUID secretKeyId = 5;
```

Comment:
Probably we can not add this as required... see details in the comment for BlockTokenSecretProto.

Reply:
Changed to optional.